### PR TITLE
Added Character repository 

### LINKED ISSUE #37: [DEV] Character Repo

### PR DESCRIPTION
**Pull request by L3viathon**

### CODE REVIEW

**Comment by L3viathon** [comment]
closes #37

**Review by pyaeger — APPROVED** [review]
Thank you.

File: src/main/java/com/amplifier/repositories/CharacterRepositoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.amplifier.repositories;
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepositoryImpl.java]: Delete this file.

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepositoryImpl.java]: Delete this file.

File: src/main/java/com/amplifier/repositories/CharacterRepository.java [hunk 2]
```diff
@@ -1,5 +1,17 @@
 package com.amplifier.repositories;
 
+import java.util.List;
+
 public interface CharacterRepository {
 
+    public List<Character> findAll();
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:15]: Annotation needed

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:15]: Annotation needed

File: src/main/java/com/amplifier/repositories/CharacterRepository.java [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+
+    public boolean create(Character character);
+
+    public Character findById(int id);
+
+    public boolean update(Character character);
+
+    public boolean deleteById(int id);
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:27]: Annotation needed

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:27]: Annotation needed

File: src/main/java/com/amplifier/repositories/CharacterRepository.java [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+
 public interface CharacterRepository {
 
+    public List<Character> findAll();
+
+    public boolean create(Character character);
+
+    public Character findById(int id);
+
+    public boolean update(Character character);
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java]: Annotation needed

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java]: Annotation needed

File: src/main/java/com/amplifier/repositories/CharacterRepository.java [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
 
+import java.util.List;
+
 public interface CharacterRepository {
 
+    public List<Character> findAll();
+
+    public boolean create(Character character);
+
+    public Character findById(int id);
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:21]: Annotation needed

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java:21]: Annotation needed

File: src/main/java/com/amplifier/repositories/CharacterRepository.java [hunk 6]
```diff
@@ -1,5 +1,17 @@
 package com.amplifier.repositories;
 
+import java.util.List;
+
 public interface CharacterRepository {
 
+    public List<Character> findAll();
+
+    public boolean create(Character character);
```
Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java]: Annotation needed

Comment (FullSol) [inline src/main/java/com/amplifier/repositories/CharacterRepository.java]: Annotation needed

**Review by FullSol — CHANGES_REQUESTED** [review]
Made a few comments for you to follow up on.

Thanks.

**Review by FullSol — APPROVED** [review]

**Pull request merged**